### DISCUSSION TITLE
Custom Model Inference

### DISCUSSION THREAD

**Discussion by kamalkannan79** [PaddlePaddle/PaddleOCR#12203] [Q&A]
from paddleocr import PaddleOCR
**custom_ocr=PaddleOCR(use_angle_cls=True,rec_model_dir="/content/drive/MyDrive/PaddleOCR/Inference_rec",det_model_dir="/content/drive/MyDrive/PaddleOCR/Inference_dir",rec_char_dict_path="/content/drive/MyDrive/PaddleOCR/ppocr/utils/en_dict.txt",use_gpu=True)**
ocr = PaddleOCR(use_angle_cls=True, lang="en")
import cv2
import matplotlib.pyplot as plt
img_path = "/content/drive/MyDrive/PaddleOCR/Data_Anno/crop_20230629_151906.jpg.jpg"
img = cv2.imread(img_path)
plt.imshow(img)
**result = custom_ocr.ocr(img)**

while inferenceing i received valueerror
**ValueError: (InvalidArgument) The size of Op(Conv) inputs should not be 0.
  [Hint: Expected in_dims[i] != 0, but received in_dims[i]:0 == 0:0.] (at /paddle/paddle/phi/infermeta/binary.cc:465)
  [operator < conv2d > error]**
  
  how to resolve this???????

**Comment by Sarveshsh8** [comment]
sure

**Comment by Liufeiran123** [comment]
same error!

**Comment by RickyGunawan09** [comment]
how u resolve this error ? i get the same error when using quick start for layout analysis

**Comment by elonzh** [comment]
PPStructure 预训练模型结构和 PaddleDetection 提供的配置文件模型结构不一样，需要使用 `deploy/python/infer.py` 进行推理，目前似乎没有提供代码方便地调用推理。
